site/content/contribute/server/developer-workflow.md

```diff
@@ -22,10 +22,15 @@ Here's a general workflow a Mattermost developer working on the [mattermost-serv
 2. On your fork, create a branch `MM-####` where #### is the ticket number if it is a [Jira](https://mattermost.atlassian.net) ticket, or `GH-####` if it is a GitHub Issue without a Jira ticket.
 3. Make the code changes required to complete your ticket, making sure to write or modify unit tests where appropriate. Make sure to use [testify](https://github.com/stretchr/testify) for new tests.
 4. To test your changes, run `make run-server` from the root directory of the server respository. This will start up the server at `http://localhost:8065`. To get changes to the server it must be restarted with `make restart-server`. If you want to test with the web app, you may also run `make run` which will start the server and a watcher for changes to the web app.
-5. Once everything works to meet the ticket requirements, stop Mattermost by running `make stop` in the server repository, then run `make check-style` to check your syntax and `make test` to run the tests.
-6. Commit your changes, push your branch and [create a pull request](https://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request).
-7. Respond to feedback on your pull request and make changes as necessary by commiting to your branch and pushing it. You might need to [rebase your changes](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) if another pull request creates conflicts.
-8. That's it! Rejoice that you've helped make Mattermost better.
+5. Once everything works to meet the ticket requirements, stop Mattermost by running `make stop` in the server repository, then run `make check-style` to check your syntax.
+6. Run the tests. You have a couple alternatives:
+     * Run `make test` to run all the tests in the project, but that may take a long time and provide very little feedback while it is running.
+     * Run individual tests by name executing `go test -run "TestName"`
```

Comment:
minor nit: looks like rest of the lines contain a period `.` It'd be good to keep it consistent by either not having it under the tests section or having them on all 4 bullet points :)